src/buza/static/buza/css/site.css

```diff
@@ -1,7 +1,7 @@
 /* Site-wide Buza styles. */
 
 .bg-buza-site {
-    background-color: #F2F2F2;
+    background-color: #FAFAFA;
```

Comment:
#F2F2F2 creates a dark background
White does not draw attention to the question cards
this light grey works well